packages/styled-components/src/models/ServerStyleSheet.js

```diff
@@ -66,7 +58,7 @@ export default class ServerStyleSheet {
     }
 
     return <style {...props} />;
-  }
+  };
 
   interleaveWithNodeStream(input: any) {
```

Comment:
I guess we do want to readd setting seal to true. But we can get away with setting it right at the top before the stream and only throwing an error here I think

Reply:
@kitten want to push onto my PR? According to the tests sealed is being set correctly in an error scenario so not 100% what you want